config.yaml

```diff
@@ -40,3 +40,8 @@ options:
       This configuration option will not be enabled on a charm upgrade, and
       cannot be toggled on in an existing installation as it will remove
       tenant access to existing buckets.
+  rbd-stats-pools:
+    type: string
+    default: ""
+    description: |
+      Enable RBD IO statistics for specified rbd pools.
```

Comment:
We seem to be lacking impl for this option?

Reply:
The configuration of rbd-stats-pools is done internally by the ceph_cos_agent module in method `mgr_config_set_rbd_stats_pools`. This is done independently of the charm (i.e. for both ceph-mon and microceph)